test/driver-test/driver-test.cpp

```diff
@@ -0,0 +1,298 @@
+/*
+* driver-test.cpp
+* Nick DeMarco, Sean Murthy
+* (c) 2020 sigcpp https://sigcpp.github.io. See LICENSE.MD
+*
+* Attribution and copyright notice must be retained.
+* - Attribution may be augmented to include additional authors
+* - Copyright notice cannot be altered
+* Attribution and copyright info may be relocated but they must be conspicuous.
+*
+* Unit tests for test driver
+*/
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <string_view>
+
+#include "../verifiers.h"
+#include "../options.h"
+#include "../tester.h"
+#include "../utils.h"
+
+void test_get_options_nominal();
+void verify_options(const Options& received, const Options& expected, const std::string& test_case);
+void test_cmd_line(const std::string& cmd_line, const Options& expected, const std::string& test_case);
+
+//unit test for the test driver
+void driver_test()
+{
+	test_get_options_nominal();
+}
+
+
+//unit test for nominal behavior of get_options
+void test_get_options_nominal()
+{
+	Options template_options;
+	template_options.command_name = "array_test";
+	Options expected_options = template_options;
+
+	//default arguments
+	std::string cmd_line("C:/Libraries/stl-lite/array_test.exe");
+	test_cmd_line(cmd_line, expected_options, "cmd-line 0");
+
+	//single name-value option supplied
+	cmd_line = "C:/Libraries/stl-lite/array_test.exe -h no";
+	expected_options = template_options;
+	expected_options.header = false;
+	expected_options.header_text = "";
+	test_cmd_line(cmd_line, expected_options, "cmd-line 1");
+
+	cmd_line = "C:/Libraries/stl-lite/array_test.exe -h yes";
+	expected_options = template_options;
+	expected_options.header = true;
+	expected_options.header_text = "Running $suite";
+	test_cmd_line(cmd_line, expected_options, "cmd-line 2");
+
+	cmd_line = "C:/Libraries/stl-lite/array_test.exe -p none";
+	expected_options = template_options;
+	expected_options.prm = pass_report_mode::none;
+	test_cmd_line(cmd_line, expected_options, "cmd_line_3");
```

Comment:
```suggestion
	test_cmd_line(cmd_line, expected_options, "cmd_line 3");
```

Reply:
Good catch @cmcmone. It should actually be `cmd-line 3`. I must have missed that one when I was changing them, thanks for pointing that out.